tangelo/toolboxes/operators/translation.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2021 Good Chemistry Company.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module to convert qubit operators to different formats."""
+
+
+from tangelo.toolboxes.operators import count_qubits, QubitOperator
+from tangelo.linq.helpers import pauli_of_to_string, pauli_string_to_of
+
+
+SUPPORTED_SOURCES = {"tangelo", "qiskit"}
+SUPPORTED_TARGETS = {"tangelo", "qiskit"}
+
+
+def translate_operator(qubit_operator, source, target, n_qubits=None):
```

Comment:
I suggest "tangelo" for the default value of `source`.

Reply:
A way to handle the 3 (4) cases (which gives you 3 different scenario in your tests). Assume source=s, target=t (s, t != "tangelo").

- Case 0: if source and target are identical, do nothing, return the input operator immediately
- Case 1: tangelo -> t
- Case 2: s -> tangelo
- Case 3: s -> t

===============

```
op = input_op # was provided as parameter to function by user
if s != "tangelo":
   op = <code to convert from source s to tangelo format>
if t != "tangelo":
  op = <code to convert from tangelo to target t format>
return op
```

Reply:
Tell me if I understood well with the changes I made. I think it is cleaner, thanks for the suggestion.